spec/spec_helper.rb

```diff
@@ -57,7 +57,7 @@
 require_relative 'support/setup_fee_provider'
 
 def setup_database
-  config = { adapter: 'sqlite3', database: 'test' }
+  config = { adapter: 'sqlite3', database: '.test' }
```

Comment:
How about use path of the DB itself to under `Dir.tmpdir`?

Reply:
Yes. I will do it.